v3/src/components/case-table/case-table-types.ts

```diff
@@ -1,5 +1,12 @@
 import {
-  CalculatedColumn, Column, EditorProps, FormatterProps, HeaderRendererProps, RowRendererProps, RowsChangeData
+  CalculatedColumn,
```

Comment:
FYI, I've never liked this kind of formatting because it wastes so much vertical real estate. I'd use something like:
```
  CalculatedColumn, CellClickArgs, Column, EditorProps, FormatterProps, HeaderRendererProps,
  RowRendererProps, RowsChangeData
```